tests/core/sys/windows/test_clipboard.odin

```diff
@@ -0,0 +1,52 @@
+//+build windows
+package test_core_sys_windows
+
+import "core:testing"
+import win32 "core:sys/windows"
+import "base:runtime"
+import "core:strings"
+import "core:mem"
+
+read_from_clipboard :: proc(wnd_handle: win32.HWND, allocator: runtime.Allocator) -> (result: string)
+{
+	if !win32.IsClipboardFormatAvailable(win32.CF_TEXT) do return;
```

Comment:
Please avoid using `do` in the Odin codebase. Prefer `{ return }` if you want to keep it a one-liner.

Reply:
Oh I didn't know that, okay I'll fix it!

Reply:
No problem. One last thing, can you remove the end of line semicolons? They're unnecessary.

The main codebase including tests is written to pass `-vet -strict-style -disallow-do`.

We have overlooked adding those switches to some of the CI tests, and so I'm adding and addressing them [here](https://github.com/odin-lang/Odin/pull/4160). But once that's finished and merged, your clipboard test will complain about the optional semicolons.

Reply:
okay done!

Reply:
Thanks. We've been quite good about it, it appears. When I added those flags to the remainder of the CI tests, it only flagged up one unused import.

Reply:
That's great! 

Btw is there a reason why `do` is not allowed? My guess is to allow people who set the `disallow-do` flag to build their code without errors, is this correct or is there some other reason?

Reply:
That is one of the main reasons, yes. The other is that it's part of the official code style.

Reply:
The problem is that `do` is occasionally very useful in constructs like `for foo in bar do if foo & 1 == 1 { ... }`.